client/src/review-components/Ratings.jsx

```diff
@@ -1,15 +1,37 @@
 import React, {useState} from 'react';
 import ReactDOM from 'react-dom/client';
 import Stars from './Stars.jsx';
+import RatingBreakdown from './RatingBreakdown.jsx';
 
 function Ratings(props) {
-  return (
-  <div className='ratings'>
-    <h1>3.5 <Stars rating='3.5'/></h1>
 
+  let testData = {
+    fiveStar: 10,
+    fourStar: 6,
+    threeStar: 4,
+    twoStar: 1,
```

Comment:
Not sure why twoStar is assigned 1 while oneStar is assigned 2

Reply:
This is the test data for number of reviews, so there's 2 one-star reviews, 1 two-star review and so on. I haven't connected the server with API data yet

Reply:
Got it!